PDF.java

```diff
@@ -0,0 +1,133 @@
+//You will need apache pdfbox, apache fontbox, apache commonsloggings, jdbc JARs
+//So far this will just generate a pdf with no values, still need to pull info from DB
+
+import java.sql.*;
+import java.io.IOException;
+import org.apache.commons.logging.*;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDPage;
+import org.apache.pdfbox.pdmodel.PDPageContentStream;
+import org.apache.pdfbox.pdmodel.font.PDFont;
+import org.apache.pdfbox.pdmodel.font.PDType1Font;
+
+public class PDF {
+	private PDDocument document = new PDDocument();
+	private PDFont font = PDType1Font.TIMES_ROMAN;
+	private PDPage page = new PDPage();
+	private ResultSet rs;
+	private Connection con;
+	private final String conURL; //URL for connecting to DB
+	private String conUSER;//Username for connecting to DB
+	private String conPW;//Password for connecting to DB
+	private String filePath;//Path to save the pdf
+	
+	public PDF(String conURL, String conUSER, String conPW, String filePath) {

```

Comment:
Why do we need connections and all that stuff? I thought we decided that we are just storing it on the local machine.

Reply:
This is to just pull data from database. Not to store the pdf.